### DISCUSSION TITLE
Export of modified field

### DISCUSSION THREAD

**Discussion by Gladbjerg** [retorquere/zotero-better-bibtex#2404] [Q&A]
Is it possible to add (perhaps postscript) the field modified. I am already exporting the tags field (https://github.com/retorquere/zotero-better-bibtex/discussions/2367#discussioncomment-4875881 )

**Comment by retorquere** [comment]
It's added automatically if you set the jabref format to anything but `no`, or you can add it in a postscript using 

```
tex.add({name: 'timestamp', value: zotero.dateModified || zotero.dateAdded})
```

**Reply by Gladbjerg** [reply]
Thank you. Is it also possible to add the item type?

**Reply by retorquere** [reply]
```
tex.add({name: 'itemtype', value: zotero.itemType})
```

**Reply by Gladbjerg** [reply]
Thank you :-)
